packages/react/src/Box/Box.tsx

```diff
@@ -29,6 +28,10 @@ type StyledBoxProps = {
   PositionProps &
   ShadowProps
 
+/**
+ * @deprecated The Box component is deprecated. Replace with a `div` or
+ * appropriate HTML element instead, with CSS modules for styling.
+ */
```

Comment:
🤔 I'm not sure what the best docs to point to are here.

Reply:
Created a doc to link to: https://github.com/primer/react/blob/hussam-i-am/deprecate-box/contributor-docs/migration-from-box.md